go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239 // indirect
 	github.com/apenwarr/fixconsole v0.0.0-20191012055117-5a9f6489cc29
+	github.com/coreos/go-iptables v0.4.5
```

Comment:
Code size impact? (we need some way to automate that report...)

Reply:
Minimal. It has no deps & is exactly the code we'd write ourselves for this.

Reply:
Also, we have an open bug for tracking binary size over time. In the other repo, iirc.